### ISSUE TITLE
android.util.AndroidRuntimeException: requestFeature() must be called before adding content

### ISSUE THREAD

**Issue by lilemma** [Nets-mobile-acceptance/Netaxept-Android-SDK#59]
After updating to newest stable versions I get this crash when opening PiaActivity.

**Pia version:** eu.nets.pia:pia-sdk:2.6.4

```
java.lang.RuntimeException: Unable to start activity ComponentInfo{com.appname/eu.nets.pia.ui.main.PiaActivity}: android.util.AndroidRuntimeException: requestFeature() must be called before adding content
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3611)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3775)
        at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:85)
        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2246)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loop(Looper.java:233)
        at android.app.ActivityThread.main(ActivityThread.java:8010)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:631)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:978)
     Caused by: android.util.AndroidRuntimeException: requestFeature() must be called before adding content
        at com.android.internal.policy.PhoneWindow.requestFeature(PhoneWindow.java:411)
        at android.app.Activity.requestWindowFeature(Activity.java:5025)
        at eu.nets.pia.ui.main.PiaActivity.onCreate(Unknown Source:4)
        at android.app.Activity.performCreate(Activity.java:8006)
        at android.app.Activity.performCreate(Activity.java:7990)
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1329)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3584)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3775) 
        at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:85) 
        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135) 
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2246) 
        at android.os.Handler.dispatchMessage(Handler.java:106) 
        at android.os.Looper.loop(Looper.java:233) 
        at android.app.ActivityThread.main(ActivityThread.java:8010) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:631) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:978) 

```
**Versions before (working):**
Kotlin 1.6.21
Compose 1.2.0-beta01
com.google.android.material:material:1.6.0-beta01
androidx.navigation:navigation-compose:2.5.0-rc01
targetSdk 31

**Versions now (not working):**
Kotlin 1.7.0
Compose 1.2.0
com.google.android.material:material:1.6.1
androidx.navigation:navigation-compose:2.5.0
targetSdk 32

**Comment by Miss-PiA** [comment]
Hi,

We have started investigating this issue and it will be resolved in the next release.

Thanks 
Miss-Pia

**Comment by Miss-PiA** [comment]
Hi @lilemma ,
Issue has been fixed in Netaxept-Android-SDK v2.7.0.
Thanks
